Content.Server/Objectives/ObjectivesSystem.cs

```diff
@@ -161,7 +166,17 @@ private void AddSummary(ref string result, string agent, List<EntityUid> minds)
                     }
                 }
             }
+
+            var successRate = totalObjectives > 0 ? (float) completedObjectives / totalObjectives : 0f;
+            agentSummaries.Add((agentSummary, successRate, completedObjectives));
         }
+
+        agentSummaries = agentSummaries.OrderByDescending(x => x.successRate)
+                                       .ThenByDescending(x => x.completedObjectives)
+                                       .ToList();
+
+        foreach (var (summary, _, _) in agentSummaries)
+            result += "\n" + summary;
```

Comment:
Same here use a stringbuilder, there's also no reason to ToList.

Reply:
Fixed